### ISSUE TITLE
[data request] YouTube-8M Segments Dataset

### ISSUE THREAD

**Issue by XinyangHan** [tensorflow/datasets#5029]
* Name of dataset: YouTube-8M Segments Dataset
* URL of dataset: https://research.google.com/youtube8m/
* License of dataset: Apache License 2.0
* Short description of dataset and use case(s): The YouTube-8M Segments dataset is an extension of the YouTube-8M dataset with human-verified segment annotations. In addition to annotating videos, we would like to temporally localize the entities in the videos, i.e., find out when the entities occur.

We collected human-verified labels on about 237K segments on 1000 classes from the validation set of the YouTube-8M dataset. Each video will again come with time-localized frame-level features so classifier predictions can be made at segment-level granularity. We encourage researchers to leverage the large amount of noisy video-level labels in the training set to train models for temporal localization.

Folks who would also like to see this dataset in `tensorflow/datasets`, please thumbs-up so the developers can know which requests to prioritize.

And if you'd like to contribute the dataset (thank you!), see our [guide to adding a dataset](https://github.com/tensorflow/datasets/blob/master/docs/add_dataset.md).

**Comment by fineguy** [comment]
Hi @XinyangHan 

Thanks for creating this request!

Our team has a limited bandwidth to implement all requested dataset. But do not hesitate to send a pull request if you already implemented this dataset.
